dev/tasks/conda-recipes/r-arrow/build_win.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+
+# Rename arrow.dll to lib_arrow.dll to avoid conflicts with the arrow-cpp arrow.dll
+sed -i -e 's/R_init_arrow/R_init_lib_arrow/g' r/src/arrowExports.cpp
```

Comment:
```suggestion
sed -i -e 's/void R_init_arrow/__declspec(dllexport) void R_init_lib_arrow/g' r/src/arrowExports.cpp
```
Maybe lookup of the *init* is failing silently, then falling back to dynamic lookup

Reply:
Massive! This worked.